pytest_fixtures/component/taxonomy.py

```diff
@@ -124,47 +137,71 @@ def module_gt_manifest_org(module_target_sat):
 @pytest.fixture(scope='session')
 def session_entitlement_manifest():
     """Yields a manifest in entitlement mode with subscriptions determined by the
-    `manifest_category.robottelo_automation` setting in manifester_settings.yaml."""
+    `manifest_category.entitlement` setting in conf/manifest.yaml."""
     with Manifester(manifest_category=settings.manifest.entitlement) as manifest:
         yield manifest
 
 
 @pytest.fixture(scope='session')
 def session_sca_manifest():
     """Yields a manifest in entitlement mode with subscriptions determined by the
-    `manifest_category.robottelo_automation` setting in manifester_settings.yaml."""
+    `manifest_category.entitlement` setting in conf/manifest.yaml."""
     with Manifester(manifest_category=settings.manifest.golden_ticket) as manifest:
         yield manifest
 
 
 @pytest.fixture(scope='module')
 def module_entitlement_manifest():
     """Yields a manifest in entitlement mode with subscriptions determined by the
-    `manifest_category.robottelo_automation` setting in manifester_settings.yaml."""
+    `manifest_category.entitlement` setting in conf/manifest.yaml."""
     with Manifester(manifest_category=settings.manifest.entitlement) as manifest:
         yield manifest
 
 
+@pytest.fixture(scope='module')
+def module_extra_rhel_entitlement_manifest():
+    """Yields a manifest in entitlementmode with subscriptions determined by the
+    'manifest_category.extra_rhel_entitlement` setting in conf/manifest.yaml."""
+    with Manifester(manifest_category=settings.manifest.extra_rhel_entitlement) as manifest:
+        yield manifest
+
+
 @pytest.fixture(scope='module')
 def module_sca_manifest():
     """Yields a manifest in Simple Content Access mode with subscriptions determined by the
-    `manifest_category.golden_ticket` setting in manifester_settings.yaml."""
+    `manifest_category.golden_ticket` setting in conf/manifest.yaml."""
     with Manifester(manifest_category=settings.manifest.golden_ticket) as manifest:
         yield manifest
 
 
 @pytest.fixture(scope='function')
 def function_entitlement_manifest():
     """Yields a manifest in entitlement mode with subscriptions determined by the
-    `manifest_category.robottelo_automation` setting in manifester_settings.yaml."""
+    `manifest_category.entitlement` setting in conf/manifest.yaml."""
     with Manifester(manifest_category=settings.manifest.entitlement) as manifest:
         yield manifest
 
 
+@pytest.fixture(scope='function')
+def duplicate_entitlement_manifest():
+    """Yields a second entitlement manifest that can be used with a function-scoped organization
+    when a test requires two separate manifests."""
+    with Manifester(manifest_category=settings.manifest.entitlement) as manifest:
+        yield manifest
+
+
+@pytest.fixture(scope='function')
+def duplicate_sca_manifest():
```

Comment:
```suggestion
def sca_manifest():
```
OR
```suggestion
def function_sca_manifest():
```
We dont say duplicate but we say about the scope is.

Reply:
Same as above my comment !